python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,22 +365,24 @@ def __get__(self, obj, cls):
             return cls._schema[name]
 
         if self._getter_name:
-            return self._bind_method(obj, self._getter_name)()
+            returnable = self._bind_method(obj, self._getter_name)()
+            if isinstance(returnable, GangaObject):
+                returnable._setParent(self)
+            return returnable
```

Comment:
It seems like sometimes `obj` is a proxy object and doing `stripProxy` didn't seem to work properly. It think the reason this got past the testing is that I didn't rerun the new tests on the release branch after merging this and somehow the legacy tests weren't hitting this (or they were and that's what was causing the LHCb problems).